test/multidep.json

```diff
@@ -0,0 +1,6 @@
+{
+  "path": "test/multidep",
+  "versions": {
+    "broccoli": ["0.16.9", "1.1.0"]
```

Comment:
This PR is backwards compatible. The integration tests execute with Broccoli 0.16.x and 1.0.x.